lib/project/index.js

```diff
@@ -113,7 +121,12 @@ module.exports = async function isInSupportWindow(projectRoot) {
               supportedRanges(info.time[info['dist-tags'].latest]),
             );
           }
-          supportChecks.push({ ...result, name, resolvedVersion });
+          supportChecks.push({
+            ...result,
+            name,
+            resolvedVersion,
+            latestVersion: info['dist-tags'].latest,
```

Comment:
will the property `dist-tags` always be present in the `info` object?

Reply:
Yeah, It must be present always.